### PR TITLE
BCNM Dropping Like Flies Fix

### PR DESCRIPTION
**Pull request by dallano**
Video and wiki link used as basis for changes:
https://ffxiclopedia.fandom.com/wiki/Dropping_Like_Flies
https://www.youtube.com/watch?v=5tm4dYla6Dg&ab_channel=mordof

Fixes:
Added lullaby resistance to houndfly
Added lullaby and sleep immunity to huntfly
Gave huntfly proper skillset (now includes sommersault) basis for this can be found in youtube video as well as wikipedia.

**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?
Video and wiki link used as basis for changes:
https://ffxiclopedia.fandom.com/wiki/Dropping_Like_Flies
https://www.youtube.com/watch?v=5tm4dYla6Dg&ab_channel=mordof

Fixes:
Added lullaby resistance to houndfly
Added lullaby and sleep immunity to huntfly
Gave huntfly proper skillset (now includes sommersault) basis for this can be found in youtube video as well as wikipedia.

## Steps to test these changes
Enter BCNM, Dropping like Flies, using sky orb in Horlais Peak.
Test mob behaviors to be accurate via using lullaby sleep / black magic sleep.

### CODE REVIEW

File: scripts/zones/Horlais_Peak/mobs/Houndfly.lua [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+local entity = {}
+
+entity.onMobSpawn = function(mob)
+    mob:setMod(xi.mod.LULLABYRES, 500)
+end
+
+entity.onMobDeath = function(mob, player, isKiller)
+end
+
+return entity
```
Comment (Mortalelite) [inline scripts/zones/Horlais_Peak/mobs/Houndfly.lua]: CI is warning you of this: All files need to end with a newline

File: scripts/zones/Horlais_Peak/mobs/Huntfly.lua [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+entity.onMobSpawn = function(mob)
+    mob:setMod(xi.mod.SLEEPRES, 1000)
+    mob:setMod(xi.mod.LULLABYRES, 1000)
+end
+
+entity.onMobDeath = function(mob, player, isKiller)
+end
+
+return entity
```
Comment (Mortalelite) [inline scripts/zones/Horlais_Peak/mobs/Huntfly.lua]: CI is warning you of this: All files need to end with a newline

**Comment by CriticalXI** [comment]
Can you add the NO_STANDBACK mod to the main Huntfly? It should close into melee distance but doesn't because its set to BLM as a main job.

**Review by zach2good — APPROVED** [review]

**Review by zach2good — CHANGES_REQUESTED** [review]
I guess waiting on:

> Can you add the NO_STANDBACK mod to the main Huntfly? It should close into melee distance but doesn't because its set to BLM as a main job.

**Review by zach2good — APPROVED** [review]

**Review by claywar — APPROVED** [review]

**Pull request merged**